pages/get_involved/support.md

```diff
@@ -9,12 +9,12 @@ menubar_toc: true
 As the US-RSE organization grows, so do the associated costs with operation.
 Running a grassroots community-driven organization takes a massive amount of effort on behalf of the community members who volunteer their time.
 
-<!--
-Donations to support US-RSE's ongoing [mission]({{ site.baseurl }}/mission/) can be made at the [US-RSE OCF member page](https://opencollective.com/usrse).
+
+Donations to support US-RSE's ongoing [mission]({{ site.baseurl }}/mission/) can be made through our [Fundrazr donation page](https://give.communityin.org/us-rse-donate?ref=ab_8CNhW0).
```

Comment:
Minor detail: `/mission/`, redirects to `/about/mission/` so you could avoid a redirect by specifying it directly.